src/components/organisms/DataTable/DataTable.tsx

```diff
@@ -88,6 +89,7 @@ const DataTable = <TData,>(
     pageSizeOptions,
     hidePagination = false,
     headComponent,
+    hidePaginationSelectionInput,
```

Comment:
Maybe we can add explicit default value since the prop itself is optional. Like TablePagination.tsx